packages/strapi-design-system/src/v2/Breadcrumbs/CrumbSimpleMenu.tsx

```diff
@@ -16,15 +15,15 @@ const StyledButton = styled(Button)`
   }
 `;
 
-export const CrumbSimpleMenu = ({ children, ...props }) => (
-  <SimpleMenu noBorder icon={<CarretDown />} as={StyledButton} size="S" {...props}>
+export interface CrumbSimpleMenuProps extends ButtonProps {
+  'aria-label': string;
+  icon?: React.ReactElement;
```

Comment:
```suggestion
```

Not sure we need this?

Reply:
otherwise the `SimpleMenu` gave me an error
<img width="835" alt="Schermata 2023-05-24 alle 11 19 49" src="https://github.com/strapi/design-system/assets/2589748/b5328f45-e21d-4ea6-bb96-f27240d1cd8a">